### ISSUE TITLE
Shuffle "about page" content

### ISSUE THREAD

**Issue by aliburchard** [zooniverse/wildcam-gorongosa#33]
HHMI is requesting the three subpages be: About, Lion Research, and Organizations, with the lion research team under Lion Research, with a mock-up of the split here: https://drive.google.com/open?id=0BxNFAhD71pdXY2l5aE1sRWtLTlhFajZFUEFRb01aRVd4enI0

I personally also think we need an additional Team page as is standard on all our projects (that includes HHMI folks and Zooniverse team on the project) as this is still a collaboration. Thoughts?

**Comment by chrissnyder** [comment]
Tentatively punting this til later, pending HHMI approval.

**Comment by DarrenMcRoy** [comment]
Just to add on, I don't support putting Zooniverse team members on the Team page. If you look at any of our old projects, you see that often 75% of the Zooniverse team members listed no longer work with Zooniverse... such things tend to become dated very quickly.

**Comment by chrislintott** [comment]
I disagree; such pages credit people who did the work to build the project. It’s important to make our team visible.

**Comment by chrissnyder** [comment]
Disagree, it's important we showcase the **science** team as it adds to the story we are trying to tell for the project. The Zooniverse team doesn't add to the narrative, gets outdated, and isn't necessary for credit as we can just point to Github.

**Comment by srallen** [comment]
Perhaps a [humans.txt](http://humanstxt.org/) file can be used to credit Zooniverse team?

**Comment by chrislintott** [comment]
I don’t think that’s right. I think it’s important to show that developers are equal partners in the project, that there is creative work that’s important (and thus credited to individuals not faceless individuals) and last that it’s important our institutions are represented.
